contracts/SecurityTokenRegistry.sol

```diff
@@ -116,10 +116,15 @@ contract SecurityTokenRegistry is ISecurityTokenRegistry, EternalStorage {
      * @notice Modifier to make a function callable only when the contract is not paused.
      */
     modifier whenNotPaused() {
```

Comment:
Can you rename to something like whenNotPausedOrOwner - `whenNotPaused` might be a bit confusing as it's a standard function with a different normal implementation.